packages/commonwealth/server/workers/sitemap/createSitemapGenerator.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { models } from '@hicommonwealth/model';
+import { expect } from 'chai';
+import { createAsyncWriterMock } from './createAsyncWriter';
+import { createDatabasePaginatorMock } from './createDatabasePaginator';
+import { createSitemapGenerator } from './createSitemapGenerator';
+
+describe('createSitemapGenerator', function () {
+  it('basic', async () => {
+    const community = await models.Community.create({ id: '1', name: 'Acme' });
```

Comment:
We should be using seed utilities in tests. This will ensure that the test DB is initialized and it will seed any dependencies (associations) with mocks for you.

See `group-lifecycle.spec.ts` for an example.

Reply:
Do you mind if I fix this with a beforeEach and afterEach where I clean up after myself rather than sticking it in group-lifecycle.spec.ts?

This way the tests are more modular and easily maintained.

Reply:
I didn't mean that you should modify `group-lifecycle.spec.ts`. I pointed it out as a good example of how to use the seed utilities. You should most definitely use `beforeEach` and `afterEach` to do setup/clean-up

Reply:
OK. I'll do a combination of both. I'll look at seed and then move them to the right pattern then use beforeEach/afterEach!  Thanks! 👍👍👍